From: Jan Kara <jack@suse.cz>
Subject: ocfs2/dlm: Fix up kABI in dlm_ctxt
References: bsc#1070404
Patch-mainline: Never, kABI

migrate_done is placed in a hole in a structure so the size of the structure
does not change. Furthermore it is for communication between ocfs2 and its DLM
so not really an external structure. Just hide the change in the structure.

Signed-off-by: Jan Kara <jack@suse.cz>

---
 fs/ocfs2/dlm/dlmcommon.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/fs/ocfs2/dlm/dlmcommon.h
+++ b/fs/ocfs2/dlm/dlmcommon.h
@@ -140,7 +140,9 @@ struct dlm_ctxt
 	u8 node_num;
 	u32 key;
 	u8  joining_node;
+#ifndef __GENKSYMS__
 	u8 migrate_done; /* set to 1 means node has migrated all lock resources */
+#endif
 	wait_queue_head_t dlm_join_events;
 	unsigned long live_nodes_map[BITS_TO_LONGS(O2NM_MAX_NODES)];
 	unsigned long domain_map[BITS_TO_LONGS(O2NM_MAX_NODES)];
